config/servicemappings/compute.yaml

```diff
@@ -2446,7 +2446,7 @@ spec:
             kind: CertificateManagerCertificate
             version: v1beta1
             group: certificatemanager.cnrm.cloud.google.com
-          valueTemplate: projects/{{project}}/locations/{{location}}/certificates/{{value}}
+          valueTemplate: projects/{{project}}/locations/global/certificates/{{value}}
```

Comment:
I just realized you updated the valueTemplate for this reference: Could you also add a test case for it?

Reply:
added test